ui/src/ensembler/components/table/ListEnsemblingJobsForEnsemblerTable.js

```diff
@@ -105,15 +106,16 @@ export const ListEnsemblingJobsForEnsemblerTable = ({
       )) : ( results.totalActiveCount > 0 && (
         <div>
           <br/>
-          <p>This Ensembler is being used by {results.totalActiveCount} <b>Active Ensembling Jobs</b>. 
-          If the ensembling jobs are in terminating state, please wait until the process complete to delete this ensembler</p>
+          This Ensembler is used by {results.totalActiveCount} <b>Active Ensembling Job{results.totalActiveCount > 1 && "s"}</b>.
+          If any ensembling jobs are in the terminating state, please wait until they are complete to delete this ensembler.
```

Comment:
Nit: `please wait until they are complete` -> `please wait until they are terminated` ?

Reply:
Sounds good! Just made that change.